src/gui/base/InfoBanner.js

```diff
@@ -0,0 +1,70 @@
+//@flow
+
+import type {AllIconsEnum} from "./Icon"
+import {Icon} from "./Icon"
+import m from "mithril"
+import {theme} from "../theme"
+import type {InfoLink, TranslationKey} from "../../misc/LanguageViewModel"
+import {lang} from "../../misc/LanguageViewModel"
+import type {ButtonParams} from "./Banner"
+import {BannerHelpLink} from "./Banner"
+import {ifNotNull} from "../../api/common/utils/Utils"
+
+export const BannerType = Object.freeze({
+	Info: "info",
+	Warning: "warning",
+})
+
+export type InfoBannerAttrs = {
+	message: TranslationKey | lazy<string>,
+	icon: AllIconsEnum,
+	helpLink?: ?InfoLink,
+	buttons: $ReadOnlyArray<ButtonParams>,
+}
+
+/**
+ * A low profile banner with a message and 0 or more buttons
+ */
+export class InfoBanner implements MComponent<InfoBannerAttrs> {
+	view(vnode: Vnode<InfoBannerAttrs>): Children {
+
+		const {message, icon, helpLink, buttons} = vnode.attrs
+
+		return m(".info-banner.flex-space-between.center-vertically.full-width.border-top.border-bottom.pt-s.pb-s.mt-s.pr-s", [
+			m(".flex.center-vertically", [
+				ifNotNull(icon, renderIcon),
+				m("small.smaller.text-break.mr", lang.getMaybeLazy(message)),
+				m("small.no-wrap.ml", buttons.map(renderButton)),
+			]),
+			ifNotNull(helpLink, renderHelpLink)
+		])
+	}
+}
+
+function renderIcon(icon: AllIconsEnum): Children {
```

Comment:
These days I tend to put them on methods so that it's easier to find but idk

Reply:
Do we rather do it that way then? We should be consistent